nucypher/network/server.py

```diff
@@ -246,23 +246,23 @@ def set_policy(id_as_hex):
             except TimeExhausted:
                 # Alice didn't pay.  Return response with that weird status code.
                 this_node.suspicious_activities_witnessed['freeriders'].append((alice, f"No transaction matching {tx}."))
-                return Response(status=402)
+                return Response(f"No paid transaction matching {tx} for this node", status=402)
 
             this_node_has_been_arranged = this_node.checksum_address in arranged_addresses
             if not this_node_has_been_arranged:
                 this_node.suspicious_activities_witnessed['freeriders'].append((alice, f"The transaction {tx} does not list me as a Worker - it lists {arranged_addresses}."))
-                return Response(status=402)
+                return Response(f"This node was not listed as servicing the policy in transaction {tx}", status=402)
```

Comment:
Gotta love the usage of 402 here.  I've waited a lifetime to use it.

> 402 Payment Required 
   Reserved for future use. The original intention was that this code might be used as part of some form of digital cash or micropayment scheme